README.md

```diff
@@ -63,10 +63,15 @@ Flags:
 Use "gtctl [command] --help" for more information about a command.
 ```
 
-Create your own GreptimeDB cluster:
+Create your own GreptimeDB cluster and etcd cluster:
 
 ```console
-gtctl cluster create mydb -n default
+gtctl cluster create mydb -n default --etcd-storage-class-name <storageClassName> --etcd-storage-size <storageSize>
```

Comment:
I think it just use the default arguments. Th command should keep it as short as possible in README.